caddyhttp/proxy/upstream_test.go

```diff
@@ -448,3 +450,42 @@ func TestHealthCheckPort(t *testing.T) {
 	})
 
 }
+
+func TestHealthCheckContentString(t *testing.T) {
+	testString := "good"
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintf(w, "blablabla%sblablabla", testString)
+		r.Body.Close()
+	}))
+
+	_, port, err := net.SplitHostPort(server.Listener.Addr().String())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer server.Close()
+
+	tests := []struct {
+		config string
+	}{
+		{"proxy / localhost:8080 {\n health_check / health_check_port " + port + " / health_check_contentstring " + testString + "\n}"},
```

Comment:
hey @tw4452852 thank for looking at it. There seem to be two types of syntax for the config file
```
proxy / localhost:8080 {
  health_check / health_check_port 1234 / health_check_contentstring teststring
}
```

and 

```
proxy / localhost:8080 {
  health_check
  health_check_port 1234
  health_check_contentstring teststring
}
```

I'm testing for both now. (they don't mix btw)